lib/auth/middleware.go

```diff
@@ -289,7 +289,8 @@ func (a *AuthMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	user, err := a.GetUser(r)
 	if err != nil {
-		trace.WriteError(w, err)
+		log.WithError(err).Warn(err.Error())
```

Comment:
Here, I'd strive to provide context even though it's not in the original code:
```go
log.WithError(err).Warn("Failed to query user.")
```
ditto below